scripts/install/opt/seagate/cortx/motr/conf/setup.yaml

```diff
@@ -17,7 +17,7 @@
 # please email opensource@seagate.com or cortx-questions@seagate.com.
 #
 
-core:
+motr:
```

Comment:
I believe this file is also used by CSM, confirm once does they also need to change anything. 
ignore if you have already checked.